ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -29,14 +29,16 @@ import org.cru.godtools.tutorial.databinding.TutorialActivityBinding
 private const val ARG_PAGE_SET = "pageSet"
 private const val ARG_FRMT_ARGS = "formatArgs"
 
+const val START_TUTORIAL = 102
```

Comment:
Not sure if this value matters or if it just needs to be unique, but at one point it was ```100``` which matched another request code so Okta login was being launched instead.

Reply:
it needs to be unique within the context of the activity or fragment. if request codes are re-used you will start getting weird behavior.

Also, the request values should be defined where you are starting activities for result, and result codes should be defined where you are returning a result.

In this case it means that this constant should be moved to `ToolDetailsFragment` because that is where you are starting the tutorial from